mgr/src/server/datastore/backup_and_restore.cr

```diff
@@ -0,0 +1,26 @@
+require "uuid"
+require "moana_types"
+require "db"
+require "sqlite3"
+
+module Datastore
+  def backup(backupdir)
```

Comment:
Accept full path similar to restore function. Rename this function to `backup_to`